tests/source/configs-fn_call_style-visual.rs

```diff
@@ -1,6 +1,12 @@
+// rustfmt-error_on_line_overflow: false
```

Comment:
Could you put the test in a new file please? I fear that adding this config option could allow a test to erroneously pass in the future.